portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase

```diff
@@ -19,58 +19,82 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-41349. Verify that a site admin could delete a Custom Export."
 	@priority = "4"
 	test DeleteCustomExportProcess {
-		LAR.exportSiteCP(
-			larFileName = "CustomExport",
-			siteScopeName = "Guest");
+		task ("Create a Custom Export") {
+			LAR.exportSiteCP(
+				larFileName = "CustomExport",
+				siteScopeName = "Guest");
+		}
 
-		LAR.deleteCurrentAndPreviousProcess(larFileName = "CustomExport");
+		task ("Delete the created Custom Export") {
+			LAR.deleteCurrentAndPreviousProcess(larFileName = "CustomExport");
+		}
 
-		AssertElementNotPresent(locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS_1");
+		task ("Assert that the Custom Export was deleted") {
+			AssertElementNotPresent(locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS_1");
+		}
 	}
 
+	@description = "This is a use case for LRQA-41349. Site admin should be able to delete a Export on the Documents and Media Portlet."
```

Comment:
Test point is that admin user can delete the portlet export process. No need to mention what portlet we used in the testcase.